WARG Onboarding Submission - Jomills Jose Anand

jomillsjoseanand opened the pull request:
Built a classification model using TensorFlow to classify the CIFAR-10 dataset. Generated a plot to check for loss and accuracy as the model trained and achieved an overall accuracy of 78.21%.

This is my onboarding submission for the WARG Computer Vision Team. Check it out and let me know in case there's anything I can change or do better. Thanks! 

![image](https://user-images.githubusercontent.com/44730936/168378538-80c5ab9e-7367-4649-804d-84dae4114704.png)

![plot](https://user-images.githubusercontent.com/44730936/168378566-0393e030-34cf-42ba-a8c5-d8bea773eb7f.png)

jomillsjoseanand committed:
Create and train CIFAR-10 classification model

Built a classification model using TensorFlow to classify
the CIFAR-10 dataset. Generated a plot to check for loss
and accuracy as the model trained and achieved an overall
accuracy of 78.21%.

rayjinghaolei commented on main.py:
```diff
@@ -1,17 +1,198 @@
+    return xTrain, xTest, yTrain, yTest
+
+
+def create_model(xTrain, yTest):
```
just a small thing here, you can only pass in the required shape for xTrain and yTest instead of passing all of them

rayjinghaolei replied:
other wise all good here, good job, this is beautiful code

jomillsjoseanand committed:
Update create_model() to only take shapes

The parameters of the create_model() function has
been modified to only take the shapes of xTrain and
yTest rather than xTrain and yTest themselves.

jomillsjoseanand commented:
I have updated the function to only accept the shapes of xTrain and yTest as arguments. Kindly check it out and provide me any further feedback. Thanks!

rayjinghaolei commented:
Good job, welcome to the team, First CV meeting is May 26th @7pm see you there
